feat(curriculum): Bash Commands quiz (#56459)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-bash-commands/66f1af4fedf643c78d024c5e.md

#### --text--

> What command is used to see the path of working directory?

Reviewer: ```suggestion
What command is used to see the path of the working directory?
```

#### --distractors--

> a) `ls`

Reviewer 2: do not give letters to the distractors or answers

Author: ok i am fixing it

---

b) `cd`

---

c) `pwd`

#### --answer--

> c) `pwd`

Reviewer 2: there are 3 distractors and one answer, total 4 choices, and they need to be all different from each other.

### --question--

#### --text--

Which command is used to see what is present in a file?

#### --distractors--

`touch`

---

`mv`

---

`ls`

#### --answer--

`more`

### --question--

#### --text--

Which command is used to create a new file?

#### --distractors--

`mkdir`

---

`cd`

---

`ls`

#### --answer--

`touch`

### --question--

#### --text--

Which command is used to remove a file?

#### --distractors--

`cp`

---

`cd`

---

`mv`

#### --answer--

`rm`

### --question--

#### --text--

Which command is used to copy a particular file to a folder?

#### --distractors--

`rm <file> <destination>`

---

`mv <file> <destination>`

---

`cd <file> <destination>`

#### --answer--

`cp <file> <destination>`

### --question--

#### --text--

Which command is used to see the list of files an folder present in a directory?

#### --distractors--

`touch`

---

`ls -l`

---

`mv`

#### --answer--

`ls`

### --question--

#### --text--

> Which will be used to rename the 'menlo.font' to 'menlo.otf'

Reviewer: ```suggestion
Which command will rename the 'menlo.font' file to 'menlo.otf'
```

#### --distractors--

`rm menlo.font menlo.otf`

---

`cp menlo.font menlo.otf`

---

`ls menlo.font menlo.otf`

#### --answer--

`mv menlo.font menlo.otf`

### --question--

#### --text--

> Which command you will be used to find things or view a file tree ?

Reviewer: ```suggestion
Which command can be used to find files or view a file tree ?
```

Author: I have tried to resolve a question issue and grammar fixes. If there is any inconsistency let me know again

#### --distractors--

`ls`

---

`mv`

---

`rm`

#### --answer--

`find`

### --question--

#### --text--

Which command is used to make a directory?

#### --distractors--

`dir`

---

`mv`

---

`touch`

#### --answer--

`mkdir`

### --question--

#### --text--

> Print 'I finished the example quiz' to the terminal using command

Reviewer: ```suggestion
Which command would print the text `I finished the example quiz`?
```

#### --distractors--

`cd I finished the example quiz`

---

`cp I finished the example quiz`

---

`ls I finished the example quiz`

#### --answer--

`echo I finished the example quiz`

### --question--

#### --text--

> Which command you will be using if you want to add a text to file directly from terminal?

Reviewer: ```suggestion
Which command would you use if you want to add text a to file directly from the terminal?
```

`rm -r`

### --question--

#### --text--

Which command is used to remove a complete directory?

#### --distractors--

`rm -r`

---

`rm`

---

`rm --help`

#### --answer--

`rmdir`

### --question--

#### --text--

Which command is used to go back to the previous directory?

#### --distractors--

`mv ..`

---

`dir ..`

---

`cp ..`

#### --answer--

`cd ..`

### --question--

#### --text--

Which flag can be used for any command to get a help menu?

#### --distractors--

`--info`

---

`--r`

---

`-l`

#### --answer--

`--help`

### --question--

#### --text--

Which command is used to close the terminal?

#### --distractors--

`close`

---

`mv`

---

`off`

#### --answer--

`exit`

### --question--

#### --text--

Which command is used to clear the terminal?

#### --distractors--

`clr`

---

`clean`

---

`clrscr`

#### --answer--

`clear`

### --question--

#### --text--

Which command is used to copy the whole folder?

#### --distractors--

`dir -r <folder_to_copy> <name_of_copy>`

---

`mv -r <folder_to_copy> <name_of_copy>`

---

`cd -r <folder_to_copy> <name_of_copy>`

#### --answer--

`cp -r <folder_to_copy> <name_of_copy>`

### --question--

#### --text--

> Which flag is used with find to search for exact folder?

Reviewer: ```suggestion
Which flag is used with find to search for an exact folder?
```

#### --distractors--

`cd -name <folder>`

---

`ls -name <folder>`

---

`mv -name <folder>`

#### --answer--

> `find -name <folder>`

Reviewer: These answers don't align with the question. The question asks about the flag, not the command.

### --question--

#### --text--

> Which flag is used with command `ls` to list the content of a folder in 'long list format;

Reviewer: ```suggestion
Which flag is used with the `ls` command to list the contents of a folder in long list format?
```

#### --distractors--

> mv

Reviewer 2: ```suggestion
`mv`
```

backticks are still needed
